pkg/addon/addon.go

```diff
@@ -318,7 +324,7 @@ func GetUIDataFromReader(r AsyncReader, meta *SourceMeta, opt ListOptions) (*UID
 		}
 		err := genAddonAPISchema(addon)
 		if err != nil {
-			return nil, fmt.Errorf("fail to generate openAPIschema for addon %s : %w", meta.Name, err)
+			return nil, fmt.Errorf("fail to generate openAPIschema for addon %s : %w (parameters: %s)", meta.Name, err, addon.Parameters)
```

Comment:
```suggestion
			return nil, fmt.Errorf("fail to generate openAPIschema for addon %s : %w (parameter: %s)", meta.Name, err, addon.Parameters)
```

We use `parameter` for cue paramter overall.